components/ProfilePage/Profile.tsx

```diff
@@ -308,17 +315,17 @@ export const ProfileHeader = ({
       <Col>
         <div>
           <ProfileDisplayName className={`overflow-hidden`}>
-            {displayName ? `${displayName}` : "Anonymous User"}
+            {displayName ? `${displayName}` : t("anonymousUser")}
           </ProfileDisplayName>
           {isOrganization ? (
             <Col>
               <Button
                 className={`btn btn-primary btn-sm py-1 ${
-                  uid ? "" : "visually-hidden"
+                  uid ? "" : t("button.hidden")
```

Comment:
I think we wouldn't need to translate this because it's a css class name that's used by the browser rather than text rendered to the user